src/main/java/in/vectorpro/dropwizard/swagger/SwaggerBundleConfiguration.java

```diff
@@ -290,10 +305,12 @@ public SwaggerConfiguration build() {
             .contact(new Contact().email(contactEmail).name(contact).url(contactUrl))
             .license(new License().name(license).url(licenseUrl))
             .termsOfService(termsOfServiceUrl);
+    oas.setInfo(info);
```

Comment:
Instead of calling setter method explicitly, please follow the same pattern being followed for Info and SwaggerConfiguration for uniform pattern in the code.

```java
final OpenAPI oas = new OpenAPI()
    .info(info)
    .servers(servers);
```

Reply:
Sure. Updated.